### PR TITLE
Enhancement theme data

### PR DESCRIPTION
**Pull request by amaury901130**
### Description
- Enhancement theme data.

For projects where the design is custom and use more colors than the ones defined for Material Design, get access to the colors depicted in the theme by using the following extension:
 * context.colors.primary
 * context.colors.secondary
 * context.colors.tertiary
 * context.colors.error
 * context.colors.neutral
 * context.colors.neutralVariant

and all their variants (v0, v10, v20, v30, v40, v50, v60, v70, v80, v90, v95, v99, v100)
 * i.e: context.colors.primary.v40
 
 Check official documentation of key Colors here:
https://m3.material.io/styles/color/the-color-system/key-colors-tones#5b4f0cd4-beb7-41b1-b6ff-a952fcbea726
https://m3.material.io/styles/color/the-color-system/color-roles
image

image
image
image

image

### CODE REVIEW

File: app/lib/presentation/themes/resources/light_theme_colors.dart [hunk 1]
```diff
@@ -74,39 +98,50 @@ class LightThemeColors implements ThemeColors {
         100: Color(0xFFFFFFFF),
       });
 
+  /**
+   * 0: shadow
+   * 99: background / surface
```
Comment (agustinkoll-rootstrap) [inline app/lib/presentation/themes/resources/light_theme_colors.dart:103]: 40 is primary ?

File: app/lib/presentation/themes/app_themes.dart [hunk 2]
```diff
@@ -39,3 +41,28 @@ class DarkTheme extends LocalTheme {
...
+ * For projects where the design is custom and don't use the Material Design colors or
+ * use more colors than the ones provided by Material Design,
+ * get access to the colors defined in the theme by using the following extension:
+ * context.colors.primary
+ * context.colors.secondary
+ * context.colors.tertiary
+ * context.colors.error
+ * context.colors.neutral
+ * context.colors.neutralVariant
+ * and all their variants (v0, v10, v20, v30, v40, v50, v60, v70, v80, v90, v99, v100)
```
Comment (agustinkoll-rootstrap) [inline app/lib/presentation/themes/app_themes.dart:55]: I think is going to be useful to have a link to the material documentation that explains every variant too

File: app/lib/presentation/themes/resources/light_theme_colors.dart (hunk 1 above)
Comment (amaury901130) [reply app/lib/presentation/themes/resources/light_theme_colors.dart:103]: yes, I have to fix these comments.

Comment (amaury901130) [reply app/lib/presentation/themes/resources/light_theme_colors.dart:103]: this is for neutral colors

File: app/lib/presentation/themes/app_themes.dart (hunk 2 above)
Comment (amaury901130) [reply app/lib/presentation/themes/app_themes.dart:55]: https://m3.material.io/styles/color/the-color-system/key-colors-tones#5b4f0cd4-beb7-41b1-b6ff-a952fcbea726
https://m3.material.io/styles/color/the-color-system/color-roles

Comment (agustinkoll-rootstrap) [reply app/lib/presentation/themes/app_themes.dart]: what do you think of adding to the code?

Comment (amaury901130) [reply app/lib/presentation/themes/app_themes.dart]: Sounds good.

**Review by agustinkoll-rootstrap — APPROVED** [review]

**Review by juanRodriguez17 — APPROVED** [review]

**Pull request merged**